app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -89,11 +97,52 @@ const LeaseUpApplicationsPage = () => {
     subStatus: null
   })
 
-  const [{ breadcrumbData }, actions] = useContext(AppContext)
   const isMountedRef = useIsMountedRef()
 
-  // grab the listing id from the url: /lease-ups/listings/:listingId
-  const { listingId } = useParams()
+  const [{ reportId, listingPreferences }, setListingState] = useStateObject({})
+  useAsyncOnMount(() => getListing(listingId), {
```

Comment:
Moved listing load into it's own asyncOnMount because we shouldn't need to check to see if we need to load the listing every time the table page changes.